drone/exec/exec.go

```diff
@@ -297,6 +301,12 @@ func exec(c *cli.Context) error {
 		})
 	}
 
+	drone_env := make(map[string]string)
+	for _, env := range c.StringSlice("env") {
+		envs := strings.Split(env, "=")
```

Comment:
instead you'll want `strings.SplitN(env, "=", 2)` which ensures the string is only split into two parts, in case the variable value has an equal sign

Reply:
Updated and tested:
```
-> ./drone exec -e FOO=bar=test
[printenv:L0:0s] + echo $FOO
[printenv:L1:0s] bar=test
```